### PR TITLE
Add BB Tool feature flag for Teacher's Digital Platform

### PR DESCRIPTION
**Pull request by dcmouyard**
Add a feature flag for the building blocks tool.

## Additions

- Feature flag for the building blocks tool
- Url to use for the building blocks tool feature flag

## Checklist

- [ ] PR has an informative and human-readable title
- [ ] Changes are limited to a single goal (no scope creep)
- [ ] Code can be automatically merged (no conflicts)
- [ ] Code follows the standards laid out in the [development playbook](https://github.com/cfpb/development)
- [ ] Passes all existing automated tests
- [ ] Any _change_ in functionality is tested
- [ ] New functions are documented (with a description, list of inputs, and expected output)
- [ ] Placeholder code is flagged / future todos are captured in comments
- [ ] Visually tested in supported browsers and devices (see checklist below :point_down:)
- [ ] Project documentation has been updated
- [ ] Reviewers requested with the [Reviewers tool](https://help.github.com/articles/requesting-a-pull-request-review/) :arrow_right:

### CODE REVIEW

File: cfgov/cfgov/urls.py [hunk 1]
```diff
@@ -397,6 +397,11 @@ def flagged_wagtail_template_view(flag_name, template_name):
     #         include_if_app_enabled('teachers_digital_platform',
     #                                 'teachers_digital_platform.search-urls')),
 
+    flagged_url('TDP_BB_TOOL',
+            r'^practitioner-resources/youth-financial-education/learn-about-the-building-blocks/take-a-tour',  # noqa: E501
+            include_if_app_enabled('teachers_digital_platform',
+                                    'teachers_digital_platform.bb-urls')),
```
Comment (Scotchester) [inline cfgov/cfgov/urls.py]: I think this will cause an error if `bb-urls.py` doesn't exist. I will confirm locally.

Comment (dcmouyard) [reply cfgov/cfgov/urls.py]: There’s currently no bb-urls.py in the current 1.0.6 release of TDP, but there should be one in the next release.

Comment (chosak) [reply cfgov/cfgov/urls.py]: [Per PEP8](https://www.python.org/dev/peps/pep-0008/#package-and-module-names), module names should never have hyphens in them. Can you please change this file (and, ideally, the other similarly named files [here](https://github.com/cfpb/teachers-digital-platform/tree/master/teachers_digital_platform)) to use underscores instead?

Comment (dcmouyard) [reply cfgov/cfgov/urls.py]: I’ll have to include that change with a bump in the TDP version with those filename changes so that nothing breaks.

**Pull request merged**